### ISSUE TITLE
[Discussion] About chinese OCR ability

### ISSUE THREAD

**Issue by mxsurui** [haotian-liu/LLaVA#706]
### Discussion

Good project indeed！

I try to use chinese handwriting OCR data train LLaVA-1.5 data like follow

![8996](https://github.com/haotian-liu/LLaVA/assets/15165983/e22278e2-9171-4039-8549-fe83b56a3e0f)

Train process the loss from 3.x to 0.2x is normal，but when i inference the data from test set, even train set, it response the very irrelevant results.

this is my infer command

```
python -m llava.serve.cli  --model-path checkpoints/llava-v1.5-13b-task-lora2  --model-base LLaVA/ckpt/llava-v1.5-13b     --image-file "/train_data/OCR/4039.jpg"
```

**Comment by XaviLv** [comment]
I don't find out the Chinese OCR ablity either after a few testing. However nice for english.

**Comment by thiner** [comment]
Yes, just tried with latest v1.6 version, it doesn't recognize Chinese characters either. As per the author's previous comment https://github.com/haotian-liu/LLaVA/issues/238#issuecomment-1760788555, I think the model is trained with English dataset only.
